package.json

```diff
@@ -36,7 +36,7 @@
     "build": "npm run types:network && npm run js:network && npm run images:network",
     "watch": "npm run images:network && rollup --watch --config rollup.config.js",
     "watch-dev": "npm run watch-dev",
-    "generate-examples-index": "node -r ./babel.mocha.js ./scripts/examples-index/generate.ts -c mynetwork -w https://visjs.github.io/vis-network -S ./scripts/examples-index/screenshot-script.js -lis",
+    "generate-examples-index": "node -r ./babel.mocha.js ./scripts/examples-index/generate.ts -t \"Vis Network Examples\" -c mynetwork -w https://visjs.github.io/vis-network -S ./scripts/examples-index/screenshot-script.js -lis",
```

Comment:
We also could use the `title` from the package.json. This could resuilt in something like: `vis-network examples`.